### PR TITLE
Add SATSolversForOcaml repository

### PR DESCRIPTION
**Pull request by oliverfriedmann**

### CODE REVIEW

File: packages/SATSolversForOcaml/SATSolversForOcaml.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,29 @@
+opam-version: "1.2"
+name: "SATSolversForOcaml"
```
Comment (avsm) [inline packages/SATSolversForOcaml/SATSolversForOcaml.0.1/opam]: any chance the capitalisation could be made more consistent, or dashes/lowercase used? The language name is "OCaml", so "SATSolversForOCaml" or "ocaml-sat-solvers" would be more natural.

**Comment by oliverfriedmann** [comment]
fair enough - will do

File: packages/SATSolversForOcaml/SATSolversForOcaml.0.1/opam (hunk 1 above)
Comment (oliverfriedmann) [inline packages/SATSolversForOcaml/SATSolversForOcaml.0.1/opam]: fair enough, will do

Comment (oliverfriedmann) [inline packages/SATSolversForOcaml/SATSolversForOcaml.0.1/opam]: done

File: packages/ocaml-sat-solvers/ocaml-sat-solvers.0.3/descr [hunk 2]
```diff
@@ -0,0 +1,5 @@
+This library contains an abstraction layer for integrating SAT
```
Comment (avsm) [inline packages/ocaml-sat-solvers/ocaml-sat-solvers.0.3/descr]: Apologies, just one last change request: the first line forms the short summary that you see when doing `opam list`, and this `descr` overflows. Do you mind shortening it, for example: "an abstraction layer for integrating SAT solvers into OCaml" would be fine.

Comment (oliverfriedmann) [inline packages/ocaml-sat-solvers/ocaml-sat-solvers.0.3/descr]: done

**Comment by avsm** [comment]
Thanks for the changes, and the new package! Since it is new, you may want to announce it on the caml-list, or on the new https://discuss.ocaml.org forum (there is a 'community' category for this).  This will help new users discover the package and ask questions about it.

**Pull request merged**